deepinv/optim/optim_iterators/optim_iterator.py

```diff
@@ -12,39 +12,36 @@ class OptimIterator(nn.Module):
     and g is a regularizer. The fixed point iteration takes the form
 
     .. math::
-        \qquad (x_{k+1}, z_{k+1}) = \operatorname{FixedPoint}(x_k, z_k, f, g, A, y, ...)
-
-    where :math:`x` is a "primal" variable converging to the solution of the minimization problem, and
-    :math:`z` is a "dual" variable.
+        \qquad x_{k+1} = \operatorname{FixedPoint}(x_k, f, g, A, y, ...)
 
+    where :math:`x` is an iterated fixed-point variable. 
 
     .. note::
-        By an abuse of terminology, we call "primal" and "dual" variables the variables that are updated
-        at each step and which may correspond to the actual primal and dual variables from 
-        (for instance in the case of the PD algorithm), but not necessarily (for instance in the case of the
-        PGD algorithm).
+        The fixed-point iterate :math:`x` does not necessarily correspond to the minimizer of :math:`F`. 
+        This is typically the case for Douglas-Rachford splitting, ADMM or primal-dual algorithms. 
+        In order to get the curent estimate of the minimizer of :math:`F` from the current iterate, one can use the function :math:`get_estimate_from_iterate`.
 
 
     The implementation of the fixed point algorithm in :meth:`deepinv.optim`  is split in two steps, alternating between
     a step on f and a step on g, that is for :math:`k=1,2,...`
 
     .. math::
-        z_{k+1} = \operatorname{step}_f(x_k, z_k, y, A, ...)\\
-        x_{k+1} = \operatorname{step}_g(x_k, z_k, y, A, ...)
+        z_{k+1} = \operatorname{step}_f(x_k, y, A, ...)\\
+        x_{k+1} = \operatorname{step}_g(z_k, y, A, ...)
```

Comment:
```suggestion
        z_{k+1} = \operatorname{step}_f(x_k, z_k, y, A, ...)\\
        x_{k+1} = \operatorname{step}_g(x_k, z_k, y, A, ...)
```

I would add it again because it is not always the case that it is written with only `x` (e.g. DRS)?